ci/WORKSPACE.filter.example

```diff
@@ -19,4 +19,4 @@ api_dependencies()
 
 load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
 go_rules_dependencies()
-go_register_toolchains()
+go_register_toolchains(go_version = "1.10.4")
```

Comment:
Do we need to modify WORKSPACE here, is there any way to hide this under a macro in `envoy_build_system.bzl`? The reason I ask this is that any change here can be chaotic to consumer of Envoy who have their own custom WORKSPACEs. Ideally we make version details as opaque as possible.